.github/workflows/tomcat_build_upload.yml

```diff
@@ -21,20 +22,42 @@ jobs:
   build-upload-artifact:
     name: Build and upload embedded Tomcat Server artifact
     runs-on: ubuntu-latest
-    needs: Trivy
+    needs: trivy
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
-      - name: Generate Artifact
+          distribution: 'adopt'
+          java-version: '1.8'
+      - name: Generate Artifact for bahmni-embedded-tomcat
         run: ./gradlew :bahmni-embedded-tomcat:clean :bahmni-embedded-tomcat:build
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.S3_MVN_ACCESS_KEY }}
           aws-secret-access-key: ${{ secrets.S3_MVN_SECRET_KEY }}
           aws-region: ap-southeast-1
-      - name: Upload Artifact
+      - name: Upload Artifact for bahmni-embedded-tomcat
         run: aws s3 cp bahmni-embedded-tomcat/build/libs/bahmni-embedded-tomcat-*.jar s3://repo.mybahmni.org/packages/build/
+
+  build-upload-artifact-10:
+    name: Build and upload embedded Tomcat 10 Server artifact
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```

Comment:
Lets use `corretto` as our distribution to keep it consistent with other repositories